src/main.rs

```diff
@@ -46,6 +46,68 @@ fn parse_matches() -> clap::ArgMatches {
                 .args(&["local_path", "release_channel"])
                 .required(true),
         )
+        // Genesis Config
+        .arg(
+            Arg::with_name("hashes_per_tick")
+                .long("hashes-per-tick")
+                .takes_value(true)
+                .default_value("auto")
+                .help("NUM_HASHES|sleep|auto - Override the default --hashes-per-tick for the cluster"),
+        )
+        .arg(
+            Arg::with_name("slots_per_epoch")
+                .long("slots-per-epoch")
+                .takes_value(true)
+                .help("override the number of slots in an epoch"),
+        )
+        .arg(
+            Arg::with_name("target_lamports_per_signature")
+                .long("target-lamports-per-signature")
+                .takes_value(true)
+                .help("Genesis config. target lamports per signature"),
+        )
+        .arg(
+            Arg::with_name("faucet_lamports")
+                .long("faucet-lamports")
+                .takes_value(true)
+                .help("Override the default 500000000000000000 lamports minted in genesis"),
```

Comment:
Can you specify `.default_value("500000000000000000")`? then you'll be able to parse this more easily.

Even better, you can pass in `&DEFAULT_FAUCET_LAMPORTS.to_string()` to avoid the copying, but it does make this function a bit goofier